104-bridge-vertx5-openapi-3.1.0.md

```diff
@@ -0,0 +1,189 @@
+# Move HTTP bridge to OpenAPI 3.1.0 with Vert.x 5
+
+This proposal recommends updating the HTTP bridge in the Strimzi project to adopt OpenAPI 3.1.0, the latest minor release in the 3.x specification series.
```

Comment:
Do we want to add a bit of context?

_OpenAPI is a standard for describing RESTful APIs using a JSON or YAML format._